### PR TITLE
#1 Build vue framework

### PR DESCRIPTION
**Pull request by roares**

### CODE REVIEW

File: src/components/HeaderBar.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      msg: {
+        type: String,
+        required: true
+      }
+    })
+    </script>
+    
+    <template>
+      <header class="header-bar">
+        <img alt="logo" class="logo" src="@/assets/logo.svg" />
```
Comment (WillemJiang) [inline src/components/HeaderBar.vue:12]: We need to consider to replace this logo.svg.

**Review by WillemJiang — COMMENTED** [review]
Is there any reason that your checked in the package-lock.json into the git repo?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Logs
+logs
+*.log
+npm-debug.log*
```
[the remark this answers is not in the document]
Comment (roares) [reply .gitignore:4]: It won't cause some side effect when keep those logs out of git repo.

File: src/components/HeaderBar.vue (hunk 1 above)
Comment (roares) [reply src/components/HeaderBar.vue]: Removed the logo.svg file.

**Comment by roares** [comment]
To lock the versions of dependencies that are installed over a project.
For more information, please checkout this link.
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

**Pull request merged**